src/reducers/posts.reducer.js

```diff
@@ -36,9 +36,16 @@ export const postsReducer = (state = initialState, action) => {
     // todo раскоментировать и имплементнуть редюсер к этому экшн типу
     //    нужно копировать state.posts и добавить к копии новый элемент, который лежит в action.payload
     //    вернуть копию стейта с подмененным массивом posts
-    // case CREATE_POST: {
-    //
-    // }
+    case CREATE_POST: {
+        const { payload: post } = action;
+        let copy = cloneDeep(state.posts);
+        copy = [...copy, post];
```

Comment:
It would be enough to use just cloneDeep for deep copying the array =) no need in copy = [...copy, post];
and you can make it constant